src/System.Numerics.Tensors/src/System/Numerics/Tensors/Experimental/IDenseTensor.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Numerics.Tensors
+{
+    public interface IDenseTensor<T> : ITensor<T>, IReadOnlyDenseTensor<T>
```

Comment:
How do you feel about the current `Tensor` classes implementing these interfaces?

Reply:
I think it might make sense, depending on how we ultimately decide to go. I don't think it is something we should do as part of the initial SxS PR, however.